### DISCUSSION TITLE
AA Torque not workin in my new Audi RS3 (2022)

### DISCUSSION THREAD

**Discussion by kthch** [agronick/aa-torque#142] [Q&A] [answered]
I installed AA Torque and gave permission in Andorid Auto.

But it's not working on the screen of my Audi (I see no AA Torque app there).

Any idea?

Thank's a lot

**Comment by BlackwoodRecordings** [comment, marked as answer]
Are you running Android 14?

**Reply by kthch** [reply]
Yes Android 14

**Reply by BlackwoodRecordings** [reply]
As mentioned in the install section "There seems to be issues with Android 14 as well."

You'll either need to downgrade to Android 13, or buy a AA dongle, as mentioned here in the discussion section.

**Reply by kthch** [reply]
Thank's

Do I need a special AA dongle or can I buy a low cost dongle?

**Reply by BlackwoodRecordings** [reply]
You'll want the special dongle. As mentioned in the discussion section here, "the AA Wireless adapter specifically
because it comes with an app that has a developer mode setting", but if you read thoroughly you'll see it didn't work for some. So I'm not totally sure it will work with your setup.

I am just keeping my phone on Android 13 and AA Torque has been working great for me since January.

**Reply by kthch** [reply]
On the Android 13 its working.
But I get now datas from some PIDs

Distance to empty: No datas
Engine Oil Temperatut: No datas
Fuel Remaining: No datas
Cost per mile: No datas

any ideas? Thanks a lot

Thomy

**Reply by BlackwoodRecordings** [reply]
Yeah different cars have different sensors so some data may not be available for everyone. I tried to dive down the rabbit hole of custom PIDs, but after a few days of feeling like I'm making zero progress I decided I'm actually fine with the limitations I have with Torque.

**Reply by kthch** [reply]
https://www.audizine.com/forum/showthread.php/920460-Torque-Non-Standard-PIDs

Maybe this helps? How I can connect one of this PIDs to AA Torque?

Thanks Thomy

**Reply by BlackwoodRecordings** [reply]
As I said, this is something that I tried to figure out but gave up on because I wasn't making progress, so do this at your own risk.


To enter a custom PID you just click the settings cog in the bottom left of the main Torque app menu screen.

Then click 'settings'

Then "Manage extra PIDs / sensors'

Then click the '3 vertical dots in top right' for options.

Click first option 'add custom PID'

Then be very careful about which signals you are sending to your car because you can mess things up if you don't actually know what you are doing.
